sstables/sstables.cc

```diff
@@ -1429,6 +1429,19 @@ size_t sstable::total_memory_reclaimed() const {
     return _total_memory_reclaimed;
 }
 
+future<size_t> sstable::reload_reclaimed_components() {
+    if (_total_memory_reclaimed == 0) {
+        // nothing to reload
+        co_return 0;
+    }
+
+    _recognized_components.insert(component_type::Filter);
```

Comment:
What will happen to reads attempting to probe the filter between this line and finishing the reload of the filter?